core/src/tangram.cpp

```diff
@@ -229,7 +229,7 @@ void applySceneUpdates() {
 
```

Comment:
Ohh wait, just saw this is in `applySceneUpdates`, which can be called multiple times from any thread, yeah `m_nextScene` copy to `scene` might not be in sync in certain cases specially if multiple updates are being performed with multiple scene files.